documentation/using-sos-private-build.md

```diff
@@ -0,0 +1,36 @@
+# Using your own SOS private build
+
+This document is written for people who need to use the bleeding edge SOS freshly built from the enlistment. You may need to do it because you want the latest feature, or you are developing your own SOS command and you want to try it out. Regardless of the reason, here is how you can do it.
+
+Before we can use the private build, of course, we must build it first. [Here](https://github.com/dotnet/diagnostics/tree/master/documentation/building) is the documentation on how to do that.
+
+## Windows
+
+On Windows, [Debugging Tools for Windows](https://docs.microsoft.com/en-us/windows-hardware/drivers/debugger/debugger-download-tools) (aka WinDBG) automatically loads a shipped version of sos.dll from the extension gallery, and we need to make sure that is undone so that we can load ours.
```

Comment:
Actually, if SOS is already loaded by the time dbgeng tries to load it from the extension gallery, the extension gallery version should NOT be loaded. 

I think that simplifies this quite a bit.  Just do the .load c:\ssd\diagnostics\artifacts\bin\Windows_NT.x64.Debug\sos.dll and "g" before coreclr is loaded.  If they wait until after coreclr.dll is loaded, then yes they will have to .unload sos.

I would also use the \<reporoot\>\artifacts\bin\Windows_NT.x64.Debug\sos.dll instead of the dotnet-sos directory you have below.  Same with the Linux plugin.  Use "artifacts/bin/Linux.x64.Debug/libsosplugin.so".